### PR TITLE
Fix log entries with user input

### PR DESCRIPTION
**Pull request by roman-khimov**
### Problem

CodeQL.

### Solution

Aquí.

### CODE REVIEW

**Comment by roman-khimov** [comment]
Huh, seems like CodeQL is not convinced.

File: pkg/rpc/server/server.go [hunk 1]
```diff
@@ -2190,3 +2193,12 @@ func validateAddress(addr interface{}) bool {
 	}
 	return false
 }
+
+func escapeForLog(in string) string {
+	return strings.Map(func(c rune) rune {
+		if c < 32 || !strconv.IsGraphic(c) {
```
Comment (AnnaShaleva) [inline pkg/rpc/server/server.go:2199]: Do we really need a separate case for `c<32`? `strconv.IsGraphic` covers it.

Comment (roman-khimov) [reply pkg/rpc/server/server.go:2199]: Well, it doesn't matter much as CodeQL is still _not_ satisfied with the fix.

**Comment by roman-khimov** [comment]
Looks a lot like github/codeql-go#635, but it should fixed in version 2.8.1 we're using.

**Review by AnnaShaleva — APPROVED** [review]

**Review by fyrchik — APPROVED** [review]

**Pull request merged**